apps/explorer/src/ui/SplitPanes.tsx

```diff
@@ -26,7 +26,7 @@ function ResizeHandle({ isHorizontal }: { isHorizontal: boolean }) {
             <div
                 data-is-dragging={isDragging}
                 className={clsx(
-                    'bg-gray-45 group-hover:bg-sui data-[is-dragging=true]:bg-hero',
+                    'bg-gray-45 hover:bg-sui data-[is-dragging=true]:bg-hero',
```

Comment:
Is this group hover removal intentional?

Reply:
🤦 removed it now. Not sure where these come from.